src/analyses/apron/relationAnalysis.apron.ml

```diff
@@ -469,10 +469,9 @@ struct
       | None -> None
       | Some st ->
         let ad = ask.f (Queries.ReachableFrom e) in
-        if Queries.AD.is_top ad then
-          None
-        else
-          Some (Queries.AD.join ad st)
+        (* See https://github.com/goblint/analyzer/issues/1535 *)
+        let ad = Queries.AD.remove UnknownPtr ad in
+        Some (Queries.AD.join ad st)
```

Comment:
Since the `None` case is now gone, this could be simplified to not have an `option` at all because it will always be `Some`.

Also, is it even necessary to remove `UnknownPtr` then? `forget_reachable` will fold over this, but only handle known addresses with `Addr` and ignore `UnknownPtr` anyway.
That's also what base does: the `UnknownPtr` is there (and warned about), but nothing is written.